man/systemd.network.xml

```diff
@@ -212,6 +212,17 @@
           below 1280 (the minimum MTU for IPv6) it will automatically be increased to this value.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ARP=</varname></term>
+        <listitem>
+          <para> A boolean. Enables or disables the ARP (low-level Address Resolution Protocol) for this interface.
+          Defaults to unset, which means that the kernel default will be used.
+
+          For example, disabling ARP is useful when creating multiple MACVLAN or VLAN virtual interfaces atop a
```

Comment:
so, if this is a bit confusing here. you format this with newlines as if these are supposed to be two paragraphs, but you do not actually place the `</para>` and `<para>` to make it two.